flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_RequestsAtis.js

```diff
@@ -161,25 +161,33 @@ class CDUAocRequestsAtis {
             store.sendStatus = "SENDING";
             updateView();
 
-            setTimeout(() => {
+            const sentRequest = () => {
```

Comment:
Not mandatory, but I'd suggest a name such as `onRequestSent`.

Reply:
Resolved